src/middlewared/middlewared/plugins/vm.py

```diff
@@ -981,12 +987,16 @@ def random_mac(self):
         """Start a VM."""
         try:
             if await self.__init_guest_vmemory(id):
-                return await self._manager.start(id)
+                # Hack until rewrite for 11.3
+                await asyncio.wait_for(self._manager.start(id), timeout=10)
```

Comment:
Isnt 10 seconds too long for the start call?

Reply:
Not in my tests, 5 wasn't long enough it doesn't block anything. Just the status updating to running.

Reply:
It will block the start call, the user will have to wait 10 seconds to know if it was successful or not.

What is taking so long? Is it the bhyve process itself?